### ISSUE TITLE
lighter secondry title

### ISSUE THREAD

**Issue by Louis-Matsika** [Louis-Matsika/kayan#14]
**make secondry text a dark grey colour(#111111?) instead of black, so that main text stands out more** 

- start with #111111
- check to see if #111111 against our current white background colour meets 7:1 contrast ratio, if not then if it at least meets 4.5:1 if doesn't meet either then pick a dark grey that does to start with.

**Comment by MaikenGunnerod** [comment]
https://github.com/Louis-Matsika/kayan/pull/16
